packages/preview-middleware-client/types/sap.suite.ui.generic.template.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'sap/suite/ui/generic/template/ListReport' {
+    import ManagedObject from 'sap/ui/base/ManagedObject';
+    export interface ListReportComponent extends ManagedObject {
+        getSmartVariantManagement: () => boolean;
+        getEntitySet: () => string;
+        getMetadata: () => ComponentMetadata;
```

Comment:
This definition already exists in parent class.

Reply:
fixed